bindings/python/capstone/__init__.py

```diff
@@ -441,6 +441,9 @@ def copy_ctypes(src):
     """Returns a new ctypes object which is a bitwise copy of an existing one"""
     dst = type(src)()
     ctypes.pointer(dst)[0] = src
+    if hasattr(src, "detail"):
```

Comment:
this line does not look right to me. i think it should be changed to something like:

    if self._cs._detail and self._raw.id != 0:

(you need to pass `_cs._detail` & `_raw.id` to `copy_ctypes()` to have this check)